returns/primitives/reawaitable.py

```diff
@@ -2,6 +2,16 @@
 from functools import wraps
 from typing import NewType, ParamSpec, TypeVar, cast, final
 
+# Try to use anyio.Lock, fall back to asyncio.Lock
+try:
+    import anyio  # noqa: WPS433
+except ImportError:
```

Comment:
I think that we might need:

```suggestion
except ImportError:  # pragma: no cover
```